ledger/stores/chunked_file_store.py

```diff
@@ -23,16 +25,27 @@ class ChunkedFileStore(FileStore):
 
     firstChunkIndex = 1
 
-    @staticmethod
-    def _fileNameToChunkIndex(fileName):
+    def _fileNameToChunkIndex(self, fileName) -> int:
         try:
```

Comment:
Please add a doc with an example of input fileName. Consider using contracts (design by contract) for checking of input here.